### PR TITLE
feat: new golanngci-lint

### PR DESCRIPTION
**Pull request by wwcchh0123**

### CODE REVIEW

**Comment by wwcchh0123** [comment]
The effects of the gofmt can be found in the following PR: https://github.com/wwcchh0123/testrepo/pull/3

**Comment by wwcchh0123** [comment]
Solve  https://github.com/qiniu/reviewbot/issues/29

File: internal/linters/go/golangci_lint/golangci_lint.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package golangci_lint
+
+import (
+	"github.com/qiniu/reviewbot/internal/linters"
+	"github.com/qiniu/x/xlog"
+)
+
+var lintName = "golangci-lint"
```
Comment (CarlJi) [inline internal/linters/go/golangci_lint/golangci_lint.go:8]: add a link  for golangci-lint official site

File: internal/linters/go/golangci_lint/golangci_lint.go [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+// refer to https://golangci-lint.run/
+var lintName = "golangci-lint"
+
+func init() {
+	linters.RegisterPullRequestHandler(lintName, golangciLintHandler)
+}
+
+func golangciLintHandler(log *xlog.Logger, a linters.Agent) error {
+	if linters.IsEmpty(a.LinterConfig.Args...) {
+		// turn off compile errors by default
```
Comment (CarlJi) [inline internal/linters/go/golangci_lint/golangci_lint.go:17]: remove this comment

File: Dockerfile [hunk 3]
```diff
@@ -34,6 +35,9 @@ RUN curl -fsSL "$GOLANG_DOWNLOAD_URL" -o golang.tar.gz \
 
 ENV PATH /usr/local/go/bin:$PATH
 
+# install golangci_lint 
+RUN curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.56.2
```
Comment (CarlJi) [inline Dockerfile:39]: This step need verification since $(go env GOPATH) may be not in system $PATH

Comment (wwcchh0123) [reply Dockerfile]: updated, move to /usr/local/go/bin/

**Pull request merged**